app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -165,6 +167,11 @@ class AttendeeViewModel(
                 mutablePendingOrder.value = it
                 orderIdentifier = it.identifier.toString()
             }, {
+                if (it is HttpException) {
+                    if (it.code() == HttpErrors.UNPROCESSABLE_ENTITY) {
+                        mutableRedirectToProfile.value = true
```

Comment:
But this error can be due to other reasons too.

Reply:
Then what should be the solution?

Reply:
Check the pointer field of the error and if it indicates unverified account error, then redirect the user to profile.

Reply:
{status: 422, source: {pointer: "/data/relationships/order"}, title: "Unprocessable Entity",…}

Reply:
And it does not have any message?

Reply:
I'm not sure if it is okay to compare the message: detail: "Unverified user cannot place free orders"

Reply:
@iamareebjamal I think there should have been more information in the pointer.

Reply:
Check if message containsIgnoreCase 'unverified user' for now. Open an issue on server to change the error to `Unauthorized` from `Unprocessable Entity` and correct the pointer. Both are wrong

Reply:
Updated